tracer/src/Datadog.Trace.Coverage.collector/AssemblyProcessor.cs

```diff
@@ -26,12 +28,12 @@ internal class AssemblyProcessor
         private static readonly object PadLock = new();
         private static readonly CultureInfo UsCultureInfo = new("us-US");
         private static readonly Regex NetCorePattern = new(@".NETCoreApp,Version=v(\d.\d)", RegexOptions.Compiled);
-        private static readonly ConstructorInfo CoveredAssemblyAttributeTypeCtor = typeof(CoveredAssemblyAttribute).GetConstructors()[0];
-        private static readonly Type ModuleCoverageMetadataType = typeof(ModuleCoverageMetadata);
-        private static readonly MethodInfo ReportTryGetScopeMethodInfo = typeof(CoverageReporter<>).GetMethod("TryGetScope")!;
-        private static readonly MethodInfo ArrayEmptyMethod = typeof(Array).GetMethod("Empty")!;
-        private static readonly MethodInfo ArrayEmptyOfIntMethod = ArrayEmptyMethod.MakeGenericMethod(typeof(int));
+        private static readonly MethodInfo ArrayEmptyOfIntMethod = typeof(Array).GetMethod("Empty")!.MakeGenericMethod(typeof(int));
         private static readonly Assembly TracerAssembly = typeof(CoverageReporter).Assembly;
+        private static readonly string[] IgnoredAssemblies =
+        {
+            "NUnit3.TestAdapter.dll"
+        };
```

Comment:
Should we ignore the other standard adapaters too? e.g. xunit, mstest?

Reply:
I'm ignoring this one because it contains the PDB file in the release. I guess is not that bad adding the others as well.

Reply:
done in [f324d97](https://github.com/DataDog/dd-trace-dotnet/pull/3170/commits/f324d97fa49eb8b13724955c2b0b3e44d19bf9da)